packages/fondue/src/components/TextInput/TextInput.tsx

```diff
@@ -63,6 +64,7 @@ export type TextInputBaseProps = {
     maxLength?: number;
     size?: number;
     spellcheck?: boolean;
+    /** @deprecated since v-beta.418, to set focus use ref exposed from the component */
```

Comment:
We don't need to mention the version here, this comment won't be visible prior to `v-beta.418`

```suggestion
    /** @deprecated a `ref` is now available so that focus can be set programmatically 
    
         @example
         ```tsx
          import { TextInput } from '@frontify/fondue';
          
         const Component = () => {
         const inputRef = useRef();
         
         useEffect(() => {
            if (inputRef.current) {
                inputRef.current.focus();
            }
         }, [)
           return <TextInput ref={inputRef} />
         }
         ```
      */
```

Reply:
Added